src/base/text/smartstring.cpp

```diff
@@ -69,8 +69,8 @@ void SmartString::trim(std::string &string, int (*ignore)(int))
 void SmartString::trimBOM(std::string &string)
 {
 	if (string.size() >= 3) {
-		if (string[0] == (char)0xEF && string[1] == (char)0xBB
-		    && string[2] == (char)0xBF) {
+		if (string[0] == static_cast<char>(0xEF) && string[1] == static_cast<char>(0xBB)

```

Comment:
This function is not used, can be removed.

Reply:
OK